packages/x-date-pickers/src/DateCalendar/tests/DateCalendar.spec.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import moment, { Moment } from 'moment';
```

Comment:
Not related to this PR, I just noticed that we had 2 spec files for this component